xchange-bitstamp/src/main/java/org/knowm/xchange/bitstamp/dto/account/WithdrawalFee.java

```diff
@@ -0,0 +1,25 @@
+package org.knowm.xchange.bitstamp.dto.account;
+
+import java.math.BigDecimal;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+import org.knowm.xchange.currency.Currency;
+
+@Getter
+@ToString
+@Builder
+@AllArgsConstructor
+public class WithdrawalFee {
+
+  private String network;
+
+  private BigDecimal fee;
+
+  private Currency currency;
+
+  public WithdrawalFee() {
```

Comment:
can be replaced with `@NoArgsConstructor`

Reply:
I'll just scrap this constructor in lieu of an immutable `WithdrawalFee` and `@Value` + `@Builder`

Reply:
Actually, that breaks the unmarshalling test: 
`Error:    WithdrawalFeesJSONTest.testUnmarshal:26 » InvalidDefinition Cannot construct instance of `org.knowm.xchange.bitstamp.dto.account.WithdrawalFee` (no Creators, like default constructor, exist): cannot deserialize from Object value (no delegate- or property-based Creator)`

Reply:
imho, you can go with `@Jacksonized` so that jackson uses builder, or also add `@NoArgsConstructor`